x-pack/plugins/security_solution/public/common/components/event_details/insights/insight_accordion.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ReactNode } from 'react';
+import React from 'react';
+import { noop } from 'lodash/fp';
+import type { EuiAccordionProps } from '@elastic/eui';
+import { EuiAccordion, EuiIcon, useGeneratedHtmlId, hexToRgb } from '@elastic/eui';
+import { euiStyled } from '@kbn/kibana-react-plugin/common';
+
+const StyledAccordion = euiStyled(EuiAccordion)`
+  border: 1px solid ${({ theme }) => theme.eui.euiColorLightShade};
+  padding: 10px 8px;
+  border-radius: 6px;
+`;
+
+const EmptyAccordion = euiStyled(StyledAccordion)`
+  background-color: rgba(${({ theme }) => hexToRgb(theme.eui.euiColorDisabled).join(',')}, 0.15);
```

Comment:
You can add the hex code for 15% transparency to the end. It might be harder for another dev to understand why that's there though, so maybe add a comment with it?

https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

Reply:
Ah, nice one. Will use that next time :)

We decided to remove the background color in the end.